ruby.asciidoc

```diff
@@ -0,0 +1,581 @@
+= Pony for Rubyists
+
+In this chapter we’ll take a look at Pony from a Ruby programmer’s
+perspective, with the assumption that the reader knows Ruby.
+
+[[what-we-like-about-ruby]]
+What we like about Ruby
+-----------------------
+
+As Ruby developers, we like Ruby’s simple design and approachable
+syntax. It has a strong focus on developer happiness and it's easy for
+newcomers to get up and running quickly.
+
+Ruby's simplicity allows developers to switch between Object-oriented
+and vaguely functional styles with ease. Blocks and dynamic typing allow
+Rubyists to be incredibly expressive in how they approach problems.
+
+Ruby also comes with a broad standard library that has provided a
+foundation for the community to build a rich ecosystem of libraries that
+cover a wide range of domains.
+
+[[how-pony-is-like-ruby]]
+How Pony is like Ruby
+---------------------
+
+Pony has a number of attributes that are similar to Ruby's, and in most
+cases a bit better too. Some of these are:
+
+* Pony is also simple and approachable
+* Pony is also object oriented, plus it blends in functional elements
+* Pony is also garbage collected, eliminating the need for developers to
+manually track memory use
+
+Other aspects of Pony feel quite a bit different than Ruby's:
+
+* Pony is strongly and statically typed, with features like algebraic
+types and traits
+* Pony’s concurrency model is a strong point, using actors with
+asynchronous behaviors that safely encapsulate all concurrent operations
+* Pony’s `ponyc` compiler is very fast, and compiles to native code
+using the LLVM backend which allows it to target just as many platforms
+if not more than Ruby
+* Ruby has a `.freeze` Object modifier that can be used on variables to
+make them unmodifiable, but it is verbose, limited and not consistently
+used. Pony uses distinct keywords, with `var` to introduce variables,
+and `let` to introduce values
+
+[[installing-pony]]
+Installing Pony
+---------------
+
+[[from-source]]
+From source
+~~~~~~~~~~~
+
+You can get the Pony source from GitHub: `git clone
+   git@github.com:ponylang/ponyc.git`
+
+Only build Pony in the directory you have cloned. If you want to install
+it, you should replace `make config=release` with `make
+   config=release` install. To build the debug version of the compiler
+`make config=debug` instead of `make config=release`.
+
+[[os-x]]
+OS X
+~~~~
+
+`$ brew update` `$ brew install ponyc`
```

Comment:
we've already had issue maintaining directions etc in different places.

this should be replaced with a link to the install instructions in the main repo README.